apps/test/unit/templates/teacherDashboard/SectionRowTest.js

```diff
@@ -132,6 +156,17 @@ describe('SectionRow', () => {
       assert.equal(col.find('a').text(), 'CS Discoveries');
     });
 
+    it('has links to both primary and secondary assignments when not editing', () => {
+      const wrapper = shallow(
+        <SectionRow
+          {...defaultProps}
+          sectionId={12}
+        />
+      );
+      const col = wrapper.find('td').at(3);
+      assert.equal(col.find('a').length, 2);
```

Comment:
It'd be nice to make some stronger assertions here - about the href and text, as you do in the test above.